src/styles/blocks/card.scss

```diff
@@ -0,0 +1,156 @@
+
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: auto 19px;
+    margin-top: 32px;
+    margin-bottom: 0;
+  }
+
+  &__p1 {
+    display: inline;
+    width: 166px;
+    color: $main-accent;
+    font-size: 12px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: auto 16px;
+    margin-top: 40px;
+    margin-bottom: 4px;
+  }
+
+  &__p2 {
+    display: inline;
+    color: $secondary;
+    font-size: 10px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0 auto;
+    margin-left: 16px;
+    margin-top: 0;
+  }
+
+  &__price-div {
+    display: flex;
+    margin: 0 16px;
+    justify-content: space-between;
+
+  }
+
+  &__price-label {
+    color: $secondary;
+    font-size: 12px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+    margin-left: 0;
+  }
+
+  &__price-amount {
+    color: $main-accent;
+    text-align: right;
+    font-size: 16px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+    margin-right: 0;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    background: $blue-accent;
+    border: transparent 1px none;
+    margin: auto 16px;
+    margin-bottom: 16px;
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    color: $white;
+    font-size: 14px;
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: auto;
+  }
+
+  &__link:hover {
+    color: $blue-accent;
+    background: $white;
+    border: #00acdc solid 1px;
+  }
+}
+
+.card:hover {
+  transform: scale(1.2);
+  transition-duration: 0.3s;
```

Comment:
please use transition on element, not on hover, if you get your cursor on the edge of  `.card` there will be little weird "jumping" - that's the reason